consensus/src/liveness/leader_reputation.rs

```diff
@@ -559,26 +566,49 @@ impl LeaderReputation {
                 )
                 .collect();
 
-                CHAIN_HEALTH_PARTICIPATING_VOTING_POWER[counter_index].set(
-                    candidates
-                        .iter()
-                        .zip(self.voting_powers.iter())
-                        .filter(|(c, _vp)| participants.contains(c))
-                        .map(|(_c, vp)| *vp as f64)
-                        .sum(),
-                );
+                let participating_voting_power = candidates
+                    .iter()
+                    .zip(self.voting_powers.iter())
+                    .filter(|(c, _vp)| participants.contains(c))
+                    .map(|(_c, vp)| *vp as f64)
+                    .sum();
+
+                CHAIN_HEALTH_PARTICIPATING_VOTING_POWER[counter_index]
+                    .set(participating_voting_power);
                 CHAIN_HEALTH_PARTICIPATING_NUM_VALIDATORS[counter_index]
                     .set(participants.len() as i64);
+
+                if chosen {
+                    // do not treat chain as unhealthy, if chain just started, and we don't have enough history to decide.
+                    result = Some(
+                        if history.len() < *participants_window_size && self.epoch <= 2 {
```

Comment:
well question is what to do if we have many governance proposals creating short epochs, or very long rounds - so that there are few in an epoch, such that 5 epochs doesn't contain enough data in the window size.

if we don't have enough events in the window size, our participation % might be smaller, but probably not by too much. (i.e. 10/30/100/300 metrics are all pretty similar  in practice

So with this - we will still allow chain backpressure. If I remove self.epoch <= 2 - we will not do a backpressure than. 

If we have chain stalls, likelihood of shorter history is higher, so I think being slightly more conservative and treating chain as unhealthy a bit earlier should be fine.